lib/diagnostics_channel.js

```diff
@@ -61,11 +116,28 @@ class ActiveChannel {
       }
     }
   }
+
+  runStores(data, fn, thisArg, ...args) {
+    this.publish(data);
+
+    // Bind base fn first due to AsyncLocalStorage.run not having thisArg
+    fn = FunctionPrototypeBind(fn, thisArg, ...args);
+
+    for (const entry of this._stores.entries()) {
```

Comment:
```suggestion
    for (const [store, transform] of this._stores.entries()) {
```

Reply:
The linter complains if you do that. I believe due to triggering the iterator interface which can be messed with.

Reply:
What about keeping the iterator as-is but changing the body a little?

```js
const [store, transform] = entry;
```